lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -140,7 +140,7 @@ class LineChartPainter extends AxisChartPainter<LineChartData>
     }
   }
 
-  void _clipToBorder(ui.Canvas canvas, ui.Size size,) {
```

Comment:
Can be reverted

Reply:
what do you mean?

Reply:
This like is changed because you've added a `,` at the end of its parameters.
These kinds of changes make confusion when looking at the git history.
That's why I'm telling you ti revert them.